lib/android/build.gradle

```diff
@@ -13,8 +13,8 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.5.3'
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:1.4.31"
+        classpath "com.android.tools.build:gradle:7.3.1"
```

Comment:
Is it ok to raise it or it would be better to keep it with previous version? Also maybe it would be better to change it to the same way its with RN 0.72?

```
        classpath("com.android.tools.build:gradle")
        classpath("com.facebook.react:react-native-gradle-plugin")
```